docs/guides/types.md

```diff
@@ -2,309 +2,282 @@
 
 This document showcases compass usage, from producing data, to using APIs for specific use-cases. This guide is geared towards adminstrators, more than users. It is meant to give the administrators an introduction of different configuration options available, and how they affect the behaviour of the application.
 
-## Prerequisites
-
-This guide assumes that you have a local instance of compass running and listening on `localhost:8080`. See [Usage](querying.md) guide for information on how to run Compass.
 
-## Creating an type
+## Prerequisites
```

Comment:
this looks similar with `querying`, should we remove this?

Reply:
I mean remove `types.md` file, or maybe we can use it to describe how we treat `type` and what are available types.

Reply:
this `types.md` is mapped to `Ingesting metadata` section in gitbook. I think we need to update the gitbook. Let me rename it to `ingestion.md` instead.